### PR TITLE
Add Playwright for Flutter Web

### PR DESCRIPTION
**Pull request by tddang-linagora**
## Issue
- https://github.com/linagora/tmail-flutter/issues/2977
- https://github.com/linagora/tmail-flutter/issues/2990

### CODE REVIEW

File: tmail_selenium/src/main/java/com/linagora/tmail/App.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.linagora.tmail;
+
+/**
+ * Hello world!
+ *
+ */
+public class App
```
Comment (chibenwa) [inline tmail_selenium/src/main/java/com/linagora/tmail/App.java:7]: Needed?

**Review by chibenwa — COMMENTED** [review]
How about `tmail_e2e` instead of `tmail_selenium` then?

File: tmail_selenium/src/main/java/com/tmail/robots/ComposerRobot.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        page.keyboard().press("Tab");
+    }
+
+    public void addContent(String content) {
+        FrameLocator frameLocator = page.frameLocator("iFrame");
+        frameLocator.locator(".note-editable").fill(content);
+    }
+
+    public void clickSend() {
+        page.getByText("Send").click();
```
Comment (dab246) [inline tmail_selenium/src/main/java/com/tmail/robots/ComposerRobot.java:39]: IMO, We should use another way to get the `Send` button. Text may change if we test in a different locale.

File: tmail_selenium/src/main/java/com/tmail/robots/MailboxDashboardRobot.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import com.tmail.base.CoreRobot;
+
+public class MailboxDashboardRobot extends CoreRobot {
+
+    public MailboxDashboardRobot(Page page) {
+        super(page);
+    }
+
+    public void openComposer() {
+        page.getByText("Compose").click();
```
Comment (dab246) [inline tmail_selenium/src/main/java/com/tmail/robots/MailboxDashboardRobot.java:13]: idem

**Comment by hoangdat** [comment]
fix for the different in `baseUrl` and `apiUrl` https://github.com/linagora/tmail-flutter/pull/3027

**Comment by quantranhong1999** [comment]
@tddang-linagora My test container work branch: https://github.com/quantranhong1999/tmail-flutter/commits/poc/selenium_testcontainer/

Do not hesitate to cherry-pick the latest commits and validate the work.

Cheers.

**Comment by quantranhong1999** [comment]
cc @tk-nguyen likely you would need to set an ENV variable to store the latest image built by the CI, so the E2E testing phase can get the image tag and run later.

**Comment by tk-nguyen** [comment]
> cc @tk-nguyen likely you would need to set an ENV variable to store the latest image built by the CI, so the E2E testing phase can get the image tag and run later.
> 
> cf [quantranhong1999@a6b3e4e#diff-a6f0c8545d1ac34ef4ea20cd3acbda7a7de3d06b17a6b660293a7f47d2272cc9R86](https://github.com/quantranhong1999/tmail-flutter/commit/a6b3e4e73164ef33f97e8b7c6e4745dd5f781c71#diff-a6f0c8545d1ac34ef4ea20cd3acbda7a7de3d06b17a6b660293a7f47d2272cc9R86)

We build an image named `tmail-web:integration-test` with each E2E flow

https://github.com/linagora/tmail-flutter/pull/2993/files#diff-b6766eb8febc0c51651250cd0cdfb44c4f0d3256470d88e62bf82fd46aa73ae0R36

**Comment by quantranhong1999** [comment]
> We build an image named tmail-web:integration-test with each E2E workflow

Ok I can hard code that image tag then.

**Pull request merged**